tests/testthat/test-tokens_chunk.R

```diff
@@ -71,19 +71,19 @@ test_that("tokens_chunk works", {
     expect_is(tokens_chunk(toks, size = 3), "tokens")
     expect_equivalent(
         as.list(tokens_chunk(toks, 3, discard_remainder = TRUE)),
-        list(c("a", "b", "c"), c("d", "e", "f"), c("a", "a", "b"))
+        list(c("a", "b", "c"), c("d", "e", "f"), c("a", "a", "b"), character())
```

Comment:
Because if you are applying a method that requires equally-sized chunks, then you don't want an unequal chunk at the end. (For the Mean Segmental TTR for instance.) For as per the example of segmenting a three-token document with `size = 5`, with `keep_remainder = FALSE` the chunk has length similar to 
```r
> 3 %/% 4
[1] 0
```

Reply:
but it might make sense to set the default to `keep_remainder = TRUE`.

Reply:
Where is the code for "Mean Segmental TTR"?

Reply:
Currently in the branch that @jiongweilua and I are working to extend `textstat_lexdiv()` to moving window tokens-based measures. We're first prototyping them, and then rewriting them (you can see the first cut in https://github.com/quanteda/quanteda/blob/MATTR/R/textstat_lexdiv_replaced.R for instance (all superceded by more efficient versions - and those we are already improving as well.).

Specific MSTTR code is:
https://github.com/quanteda/quanteda/blob/MATTR/R/textstat_lexdiv.R#L419-L438

Ultimately this could all be done nicely in C++ but would be even neater to generalize the workhorses of these functions by adding a `tokapply()` that operated like lapply but over tokens including flexible windows or samples within documents' tokens.  Here one of those is in R:
https://github.com/quanteda/quanteda/issues/1519#issuecomment-446434372.

Reply:
If you split a vector by a factor with all levels (document IDs), you have no problem with documents being removed by `tokens_chunk()`. I will remove `keep_remainder` argument.

Reply:
Yes but the point is that the length of the split elements should be the same. That does not happen when some segments are remainders. Let's keep `keep_remainder` but set the default to `TRUE`.

Reply:
Why don't you do it using `lengths()`, which is pretty fast?

Reply:
You can also let `textstat_lexdiv(x, "MTTR")` to return NA for document shorter than a certain size.

Reply:
Yes it should but it also should not average in segments shorter than the specified size. 

Anyway more generally on chunks: it’s fundamentally an integer division operation, and integer divisions have a quotient and a modulus. The keep_remainder is to keep the modulus. Why would we not provide that option?

Reply:
I don't think we need that because I cannot imagine any other use-case, and it can be done just by `textstat_lexdiv(toks[lengths(toks) >= window], "TTR")`.